.idea/compiler.xml

```diff
@@ -4,9 +4,8 @@
     <annotationProcessing>
       <profile default="true" name="Default" enabled="true" />
     </annotationProcessing>
-    <bytecodeTargetLevel>
+    <bytecodeTargetLevel target="1.8">
```

Comment:
Try to do not have such .idea changes in a PR.
If they are needed, create a new PR, so that purpose of it is clear.